Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -230,10 +228,7 @@ protected virtual void RegisterManagers()
             InputManager.Instance.InputEnabled += OnInputEnabled;
             InputManager.Instance.InputDisabled += OnInputDisabled;
 
-            if (FocusManager.IsInitialized)
```

Comment:
Correct.  FocusManager is a required component just as much as InputManager is, so if we somehow end up with a scene without a FocusManager in it, we want a big loud error just like we get when InputManager is missing so we can fix it.